scripts/feature_tests/upgrade/upgrade_vars.sh

```diff
@@ -10,7 +10,7 @@ export CAPI_VERSION="${CAPI_VERSION:-v1alpha4}"
 export CAPI_REL_TO_VERSION="v1.0.1"
```

Comment:
should not this be the latest

Reply:
+1 also [CAPM3_REL_TO_VERSION](https://github.com/metal3-io/metal3-dev-env/blob/main/scripts/feature_tests/upgrade/upgrade_vars.sh#L13) needs to be updated

Reply:
Done